test/index.js

```diff
@@ -116,6 +116,20 @@ describe('PWMetrics', () => {
         });
       });
     });
+
+    describe('with one run with expectations', () => {
+      beforeEach(() => {
+        const {opts} = runOptions.startWithOneRunWithExpectations;
+        pwMetrics = new PWMetrics(runOptions.startWithOneRun.url, opts);
```

Comment:
You should use `url` from the same fixture here, right? `startWithOneRunWithExpectations`